autovalet_lane_detection/src/autovalet_lane_detection/LaneDetector.py

```diff
@@ -175,11 +176,11 @@ def center_line_detection(self, img):
             # assuming slope being positive, might have bug
             # sign of slope is flipped here b/c origin is at top-left corner of image
             if slope > 1:
-                coordinates = [[X1+i, int(Y1-i*slope)] for i in range(X2-X1)]
-            else:
+                # coordinates = [[X1+i, int(Y1-i*slope)] for i in range(X2-X1)]
```

Comment:
get rid of commented lines we're not using

Reply:
done